.github/workflows/release.yml

```
@@ -4,30 +4,30 @@ on:
   push:
     branches:
       - master
-      - feature/query
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@v4
         with:
           # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
           fetch-depth: 0
 
       - name: Setup Node.js
-        uses: actions/setup-node@v2.1.2
+        uses: actions/setup-node@v4
         with:
-          node-version: 14
+          node-version: 20
 
       - name: Install Dependencies
         run: yarn
```

```suggestion
          node-version: 20

      - name: Install Dependencies
        run: npm i
```

Don't we need to also amend this?

---

Of course!